frontend/server/src/Controllers/Session.php

```diff
@@ -599,8 +607,15 @@ private static function ThirdPartyLogin(
                 throw new \OmegaUp\Exceptions\NotFoundException('userNotExist');
             }
         }
+        if (is_null($identity->username)) {
+            throw new \OmegaUp\Exceptions\NotFoundException('userNotExist');
+        }
+        $user = \OmegaUp\DAO\Users::FindByUsername($identity->username);
```

Comment:
```suggestion
        $user = \OmegaUp\DAO\Users::getByPK($identity->user_id);
```

claro que esto sólo es posible si la identidad tiene un usuario asociado (pero... debería, no? entonces está bien arrojar una excepción si `is_null($identity->user_id)`).

Reply:
Listo